admin/tools/docker/replication/container/dev/Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN curl -sO http://www.slac.stanford.edu/exp/lsst/qserv/2016_09/_downloads/qser
 RUN chmod +x qserv-install-deps-rhel7.sh
 RUN /bin/bash qserv-install-deps-rhel7.sh
 
+RUN yum --assumeyes install libuuid-devel
```

Comment:
This extra dependency is required by Stack package antlr4. Otherwise the LSST Stack wouldn't be built when building this development container.